misc/python/materialize/cli/mzconduct.py

```diff
@@ -1272,6 +1309,18 @@ def mzcompose_up(
     return spawn.runv(cmd + services, env=_merge_env(extra_env))
 
 
+def mzcompose_restart(
+    services: List[str],
+    args: Optional[List[str]] = None,
+    extra_env: Optional[Dict[str, str]] = None,
+) -> subprocess.CompletedProcess:
+    if args is None:
+        args = []
+    cmd = ["bin/mzcompose", "--mz-quiet", *args, "restart"]
+    print("CWD:", os.getcwd())
```

Comment:
I think this print should be removed (https://github.com/MaterializeInc/materialize/pull/4993)

Reply:
Done.